server/app/models/user.rb

```diff
@@ -41,7 +41,27 @@ def attending_meetings
   def pending_meetings
     ret = []
     for user_meeting in user_meetings
-      if !user_meeting.accepted
+      if user_meeting.status == "pending"
```

Comment:
Pretty sure this can be something like `user_meetings.where(status: "pending")` , but this works too. Same for the other funcs

Reply:
Oh yeah I was about to change it to one liner but didn't want to bloat the PR. but it looks like it's already bloated so I'll change this one lol.